doc/nrf/app_memory.rst

```diff
@@ -62,19 +68,28 @@ Complete the following actions to optimize the Bluetooth® part of your applicat
 
 For reference, you can find minimal footprint configurations of the :ref:`peripheral_lbs` sample in :file:`nrf/samples/bluetooth/peripheral_lbs/minimal.conf` and the :ref:`peripheral_uart` sample in :file:`nrf/samples/bluetooth/peripheral_uart/minimal.conf`.
 
+.. _app_memory_matter:
 
-Thread
+Matter
 ******
 
-Complete the following actions to optimize the Thread part of your application:
+Besides applying `General recommendations`_, you can also complete the following actions to optimize the :ref:`Matter <ug_matter>` part of your application:
 
-* Disable features that your application does not use.
-  For example, disable the following features:
+* Make sure Zephyr's :ref:`zephyr:shell_api` are disabled for your application.
+  These are listed in a dedicated section in each Matter sample's :file:`prj.conf` file.
```

Comment:
Maybe it's worth adding that for each sample there is a [prj_release.conf](https://github.com/nrfconnect/sdk-nrf/blob/main/samples/matter/light_bulb/configuration/nrf52840dk_nrf52840/prj_release.conf) with can be used as a first aid to reduce the footprint.